packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6049,22 +6049,66 @@ class Semantics extends SingleChildRenderObjectWidget {
 /// A widget that merges the semantics of its descendants.
 ///
 /// Causes all the semantics of the subtree rooted at this node to be
-/// merged into one node in the semantics tree. For example, if you
-/// have a widget with a Text node next to a checkbox widget, this
-/// could be used to merge the label from the Text node with the
+/// merged into one node in the semantics tree.
+///
+/// If you have a widget with a Text node next to a checkbox
+/// widget, this could be used to merge the label from the Text node with the
 /// "checked" semantic state of the checkbox into a single node that
 /// had both the label and the checked state. Otherwise, the label
 /// would be presented as a separate feature than the checkbox, and
 /// the user would not be able to be sure that they were related.
 ///
+/// {@tool snippet --template=stateful_widget_scaffold}
+///
+/// To see the end result, make sure to set MaterialApp's showSemanticsDebugger
```

Comment:
`[MaterialApp.shoeSemanticsDebugger]` to get a cross-link?

Also `[CupertinoApp]` and `[WidgetApp]`